packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -450,6 +454,79 @@ Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by
       }
     });
 
+    testWithoutContext('ffi failure on ARM macOS prompts gem install', () async {
+      pretendPodIsInstalled();
+      fileSystem.file(fileSystem.path.join('project', 'ios', 'Podfile'))
+        ..createSync()
+        ..writeAsStringSync('Existing Podfile');
+
+      when(mockProcessManager.runSync(<String>['sysctl', 'hw.optional.arm64']))
```

Comment:
Adding this file to my mental list to swap to `FakeProcessManager`.  Make the new tests match the rest of the file, for now.